.config/dictionaries/technical.dic

```diff
@@ -56,6 +57,7 @@ assetsinfo
 autofetch
 autorenew
 azuretools
+bbcccccc
```

Comment:
Same, why are we adding this?

Reply:
These are some docs coming from `sp-core` talking about bit manipulations and prefixes in `from_ss58check_with_version`